src/common/utility/configfile.cpp

```diff
@@ -601,10 +604,13 @@ void FConfigFile::LoadConfigFile ()
 	FileReader file;
 	bool succ;
 
-	FileExisted = false;
+	if (!std::filesystem::exists(PathName.GetChars()))
+		FileExisted = false;
```

Comment:
You changed a guaranteed initialization here to a conditional one. I don't like this. Is this defined elsewhere?

Reply:
Yes, it's set to true in the constructor. It doesn't compile on macOS and gcc 7.